src/SFA.DAS.ApplyService.InternalApi.UnitTests/OversightControllerTests.cs

```diff
@@ -97,5 +97,40 @@ public async Task Check_pending_results_are_as_expected()
             Assert.AreEqual(oversight.ApplicationReferenceNumber, returnedOversight.ApplicationReferenceNumber);
             Assert.AreEqual(oversight.OversightStatus, returnedOversight.OversightStatus);
         }
+
+
+        [Test]
+        public async Task Check_oversight_details_result_is_as_expected()
+        {
+            var applicationId = Guid.NewGuid();
+
+            var oversight = new ApplicationOversightDetails
+            {
+                Id = Guid.NewGuid(),
+                ApplicationId = applicationId,
+                OrganisationName = "XXX Limited",
+                Ukprn = "12344321",
+                ProviderRoute = "Main",
+                ApplicationReferenceNumber = "APR000111",
+                OversightStatus = "New"
+            };
+
+            _mediator
+                .Setup(x => x.Send(It.IsAny<GetOversightDetailsRequest>(), It.IsAny<CancellationToken>()))
+                .ReturnsAsync(oversight);
+
+            var actualResult = await _controller.OversightDetails(applicationId);
+
+            var returnedOversight = actualResult.Value;
+
+            Assert.AreEqual(oversight.Id, returnedOversight.Id);
```

Comment:
In contract to my previous comment, these checks are probably overkill.  IsSame would be better and more appropriate to the code in the controller.